tests/envs/test_grid_row.py

```diff
@@ -243,5 +244,6 @@ def test_grid_row_door():
         assert option.terminal(state)
         if all(a.holds(state) for a in ground_nsrt.add_effects):
             break
+    env.render_state(state, task)
```

Comment:
nit: instead of just calling `render_state`, you should assert that `render_state` isn't none and actually returns what it says it's returning (i.e. an array of some kind).